dataParser.py

```diff
@@ -33,4 +33,25 @@ def parse_section_information(self, section_information=""):
         if ("tut_id" not in values):
             values["tut_id"] = None               
             
-        return values
\ No newline at end of file
+        return values
+    
+    def parse_days(self,section_information=""):
+        days_dict ={
+            "Mon":"Monday",
+            "Tue":"Tuesday",
+            "Wed":"Wednesday",
+            "Thu":"Thursday",
+            "Fri":"Friday",
+            "Sat":"Saturday",
+            "Sun":"Sunday",
+        }
+
+        days_arr = section_information.split(" ")
+
+        #This is at most O(7) because only seven days in a week
+        for x in range(len(days_arr)):
```

Comment:
NIT: You can do `for day in days_arr`, this will save computation time because you won't have to access the `x`th index everytime.